src/main/resources/db/migration/V42_3__Create_user_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+
+CREATE TABLE IF NOT EXISTS "user" (
+   id VARCHAR DEFAULT UUID_GENERATE_V4(),
+   firstname VARCHAR(50),
+   lastname VARCHAR(50),
+   mail VARCHAR(50) NOT NULL,
+   birthdate DATE,
+   role VARCHAR(50),
+   creation_datetime TIMESTAMP,
+   last_update_datetime TIMESTAMP,
+   PRIMARY KEY(id)
```

Comment:
it's true, but for the view, it's better to put it directly on top.

Reply:
okay